hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/RocksDatabase.java

```diff
@@ -665,6 +666,32 @@ public Collection<ColumnFamily> getExtraColumnFamilies() {
     return Collections.unmodifiableCollection(columnFamilies.values());
   }
 
+  public void dropColumnFamily(String tableName) throws RocksDatabaseException {
+    ColumnFamily columnFamily = columnFamilies.get(tableName);
+    if (columnFamily != null) {
+      try {
+        getManagedRocksDb().get().dropColumnFamily(columnFamily.getHandle());
```

Comment:
Shouldn't it increment reference count via `acquire()` while operating on the DB?